<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="MainViewWidget" name="myOpenGLWidget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>491</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="rotXSlider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSlider" name="rotYSlider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSlider" name="rotZSlider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>xPos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>yPos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>zPos</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>9</y>
     <width>261</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layout"/>
  </widget>
  <zorder>rotXSlider</zorder>
  <zorder>rotYSlider</zorder>
  <zorder>rotZSlider</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>myOpenGLWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainViewWidget</class>
   <extends>QWidget</extends>
   <header>mainviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
